### ISSUE TITLE
Support for detecting if an object is "connected" to Qml in some way.

### ISSUE THREAD

**Issue by pauldotknopf** [pauldotknopf/net-core-qml#14]
```c#
public static class Qml
{
    static bool IsConnected(this object instance)
    {
        // Return true if the Qml is currently alive in Qml in any way, otherwise false.
    }
}
```

**Comment by pauldotknopf** [comment]
This should be done before #7.

This will require us to track .NET instances that are currently alive in the Qml world.

This is the rough idea:

We store the instances in .NET with a ```Dictionary<object, NetInstance>```. When sending an object to Qml, we need to check if an existing ```NetInstance``` already exists in the dictionary and use it, otherwise create one. Within the deconstructor of ```NetInstance```, we need to reach out to the .NET world and remove the instance from our global dictionary.

```IsConnected``` will simple do a ```instance.ContainsKey``` to determine if object is currently wrapped in Qml/V8 somewhere.

**Comment by devmil** [comment]
no longer needed?

**Comment by pauldotknopf** [comment]
Yeah, not sure the use case anymore. I was initially only doing to do this to ensure the underlying facilities are in place for doing signals, but that was solved on your end.
